### ISSUE TITLE
Ensures aria-hidden elements are not focusable nor contain focusable elements (i[role="none"][data-tabster-dummy=""]:nth-child(1)): A11y_Awesome AZD_Landing page_Top navigation_AI4W.

### ISSUE THREAD

**Issue by Harika1702** [Azure/awesome-azd#341]
### Tags
#A11yTCS; #A11ySev2; #A11yMAS; #WCP; #CELA P3; #ChromiumEdge; #FTP; #Win11; #awesome-azd-wcp; #CELA_AwesomeAZD_Web_Mar2024; #MAS4.1.2; #Name Role Value; #DesktopWeb; #AI4W; #A11yAuto;

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### Environment Details:
Application: Awesome AZD
URL: https://azure.github.io/awesome-azd/
Chromium Edge: Version 123.0.2420.20 
Windows 11 Enterprise.
Version 23H2 OS build 22631.2050.
### Repro Steps: 

1. Hit this URL: https://azure.github.io/awesome-azd/.
2. Now run AI4W.
3. verify the issue that Ensures aria-hidden elements are not focusable nor contain focusable elements.
### Issue: 
Ensures aria-hidden elements are not focusable nor contain focusable elements (aria-hidden-focus - https://accessibilityinsights.io/info-examples/web/aria-hidden-focus)
### Target application: 
Awesome Azure Developer CLI - https://azure.github.io/awesome-azd/
### Element path:
 i[role="none"][data-tabster-dummy=""]:nth-child(1)
### How to fix: 
Fix all of the following:
  Focusable content should have tabindex='-1' or be removed from the DOM
### Environment: 
====
This accessibility issue was found using Accessibility Insights for Web 2.41.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
### Recommendations:
Refer [Developer Resources (sharepoint.com).](https://microsoft.sharepoint.com/sites/accessibility/SitePages/Developer-Resources.aspx)
### MAS reference:
[MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)
